package.json

```diff
@@ -13,11 +13,13 @@
     "preview": "vite preview",
     "deploy": "vite build --base=/nucmorph-colorizer/ && gh-pages -d dist",
     "lint": "eslint --config ./.eslintrc --ext .ts --ext .tsx --ext .js --ext .jsx .",
+    "lint-fix": "eslint --rule \"simple-import-sort/imports: warn\" --rule \"simple-import-sort/exports: warn\" --fix --config ./.eslintrc --ext .ts --ext .tsx --ext .js --ext .jsx .",
```

Comment:
does this mean that lint-fix will ONLY do the import resorting?

Reply:
I think I had the rules disabled in linting for a sec there, because prettier and the eslint plugin were fighting. BUT I finally figured out the setting to get them to play nice! Now this does the same thing as regular linting, just adding the `--fix` flag.